.gitignore

```diff
@@ -64,6 +64,7 @@ phpunit.xml
 # Composer
 #-------------------------
 vendor/
+composer.lock
```

Comment:
Please remove. Lock files should be version controlled to make sure everyone gets the same versions of packages by default, since they are known to work.

Reply:
here is little complicated, because I am using PHP 8.2 on my local machine, so if I commit composer.lock it will include dependencies installed for PHP 8.2. 
But you are right about locking version control, so instead commit composer.lock I can lock versions in composer.json.

Reply:
We can deal with it like this for now. Thanks.